pkg/controller/realmadmin/show.go

```diff
@@ -66,7 +66,15 @@ func (c *Controller) HandleShow() http.Handler {
 			return
 		}
 
-		c.renderShow(ctx, w, realm, stats, userStats)
+		if controller.AcceptsType(r, controller.ContentTypeJSON) {
+			resp := struct {
```

Comment:
This should either be a top-level struct documented in the `api` package or at least defined outside of the body so it's not re-allocated each time.